packages/vault/src/objects/objectsStore.ts

```diff
@@ -0,0 +1,68 @@
+import { produce } from 'immer'
+import { StateCreator } from 'zustand'
+
+import { matchesQuery } from '../utils/utils'
+import { initialObjectState, ObjectStore } from './objectsState'
+
+export const objectSlice: StateCreator<ObjectStore> = (set, get) => ({
+  objects: {},
+  ensureObject: (objectName) => {
+    set(
```

Comment:
I think set returns `void`, but usually the tendency is to return it or write lambda like `ensureObject: (objectName) => set(`